assignment/index.html

```diff
@@ -62,19 +62,42 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
-
-      /* =====================
-
-      End code
-
-      ===================== */
-
-
-      jsonToCsv(healthCenters);
+      let new_array=[['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']]
+        for (var i=1; i<healthCenters.length; i++) {
+          var x = [parseFloat(`${healthCenters[i].LNG}`),
+                   parseFloat(`${healthCenters[i].LAT}`),
+                      `${healthCenters[i].OBJECTID}`,
+                      `${healthCenters[i].NAME}`,
+                      `${healthCenters[i].ORGANIZATION}`,
+                   Number(`${healthCenters[i].ZIP}`),
+                      `${healthCenters[i].PHONE}`,
+                      `${healthCenters[i].WEBSITE}`,
+                      `${healthCenters[i].DENTAL_PHONE}`,
+                      `${healthCenters[i].FULL_ADDRESS}`];
+          new_array.push(x)
+        };
+      console.log(new_array)
+      var Icon1 = L.icon({
+          iconUrl: 'F:/BACKUP/CPLN611JavaScript/cpln692-week2/assignment/images/marker-icon-black.png',
```

Comment:
Absolute path URLs are best avoided because they make it difficult to use this code on anyone elses computer.

If you turn it relative, and do this:

`iconUrl: 'images/marker-icon.png'`

it works anywhere.